test/AlarmClockTest.cpp

```diff
@@ -35,24 +29,7 @@ namespace {
       return std::chrono::duration_cast<milliseconds>(t).count();
    }
 
-   template <typename Duration>
-   int GetTimingLeeway(Duration t) {
-
-      auto timeInMicro = ConvertToMicroSeconds(t);
-      auto k500MsInMicro = ConvertToMicroSeconds(milliseconds(500));
-
-      // One sleep of 500ms or less => 200us leeway
-      if (timeInMicro <= k500MsInMicro) {
-          return kTimingLeeway;
-      // One leeway of 200us for every loop of 500ms, plus
-      //    one extra for the leftover time    
-      } else if (timeInMicro % k500MsInMicro != 0) {
-         return kTimingLeeway * (timeInMicro / k500MsInMicro + 1);
-      // One leeway of 200us for every loop of 500ms
-      } else {
-         return kTimingLeeway * (timeInMicro / k500MsInMicro);
-      }
-   }
+   void FakeSleep(unsigned int ignored) {}
```

Comment:
These tests would be better if you saved the value FakeSleep was called with, and you then compared that to the value to are using in the unit tests.